collection-scripts/gather

```diff
@@ -18,7 +18,7 @@ resources+=(clusteroperators)
 resources+=(certificatesigningrequests)
 
 # Machine/Node Resources
-resources+=(nodes machineconfigs machineconfigpools)
+resources+=(nodes machineconfigs machineconfigpools kubeletconfigs containerruntimeconfigs)
```

Comment:
if https://github.com/openshift/machine-config-operator/pull/1779 works, I think we'll drop everything here except `nodes`

Reply:
pushed a fix!